README.md

```diff
@@ -40,131 +40,152 @@ FIEDLER, Niklas, et al. [ImageTagger: An Open Source Online Platform for Collabo
+    ```
+
+3.  #### Configuring ImageTagger to connect to the database
+    
+    Please see the lower section about application configuration on how to configure ImageTagger for your specific
+    database credentials.
+   
+4.  #### Initialize the database
+    
+    Run the following to create and setup all necessary database tables:
+    ```shell
+    ./manage.py migrate
+    ```
+
+5.  #### Create a user
+    
+    ```shell
+    export DJANGO_CONFIGURATION=Prod
+    ./manage.py createsuperuser
+    ```
+
+6.  #### Run the server
+    
+    ```shell
+    export DJANGO_CONFIGURATION=Prod
+    ./manage.py runserver
+    ```
+   
+**In a production deployment** it is necessary to run the following commands after each upgrade:
+```shell
+export DJANGO_CONFIGURATION=Prod
 ./manage.py migrate
 ./manage.py compilemessages
 ./manage.py collectstatic
 ```
 
-Our production uwisgi config can be found at https://github.com/fsinfuhh/mafiasi-rkt/blob/master/imagetagger/uwsgi-imagetagger.ini
-
-Example Nginx Config:
-
-```
-server {
-        listen 443;
-        server_name imagetagger.bit-bots.de;
-
-        ssl_certificate /etc/letsencrypt/certs/imagetagger.bit-bots.de/fullchain.pem;
-        ssl_certificate_key /etc/letsencrypt/certs/imagetagger.bit-bots.de/privkey.pem;
-        include /etc/nginx/ssl.conf;
-        include /etc/nginx/acme.conf;
-        ssl on;
-
-        client_max_body_size 4G;
-
-        access_log /var/log/nginx/imagetagger.bit-bots.de.access.log;
-        error_log /var/log/nginx/imagetagger.bit-bots.de.error.log;
-
-        location /static {
-                expires 1h;
-                alias /var/www/imagetagger;
-        }
-
-        location /ngx_static_dn/ {
-                internal;
-                alias /srv/data/imagetagger/storage/pictures/;
-        }
-
-        location / {
-                include uwsgi_params;
-                uwsgi_pass 127.0.0.1:4819;
-                uwsgi_read_timeout 120;
-        }
-}
-```
-
-If you want to provide zip files of image sets, set `ENABLE_ZIP_DOWNLOAD = True` in your `settings.py`.
-A daemon that creates and updates the zip files is necessary, you can start it with `./manage.py runzipdaemon`.
-Please take into account that the presence of zip files will double your storage requirement.
-
-Zip archive download via a script is also possible. The URL is `/images/imageset/<id>/download/`. A successful request
-returns HTTP 200 OK and the zip file. When the file generation is still in progress, HTTP 202 ACCEPTED is returned.
-For an empty image set, HTTP 204 NO CONTENT is returned instead of an empty zip archive.
+for additional steps on some releases see instructions in [UPGRADE.md](https://github.com/bit-bots/imagetagger/blob/master/UPGRADE.md)
+
+### In-Docker
+
+1.  #### Checkout the latest release
+    
+    ```shell
+    git checkout v…
+    ```
+
+2.  #### Build the container image
+    
+    *Note:* the Dockerfile has been tested with [podman](https://podman.io/) as well.
```

Comment:
Is this really relevant?

Reply:
True it is a bit unnecessary but I have previously run across some Dockerfiles which were not compatible.